lib/std/os/linux.zig

```diff
@@ -7175,7 +7175,7 @@ pub const perf_event_attr = extern struct {
     /// Defines size of the user stack to dump on samples.
     sample_stack_user: u32 = 0,
 
-    clockid: clockid_t = .REALTIME,
+    clockid: clockid_t = 0,
```

Comment:
Was that meant to be part of the changes?

Reply:
This seems to be an old commit that wasn't included in the merged PR.

Reply:
Yes, I noticed it in diff and removed the change